src/components/custom-media-library/CustomMediaLibrary.test.tsx

```diff
@@ -215,6 +230,26 @@ describe(CustomMediaLibraryUI.name, () => {
       expect.objectContaining({ url: testItem.url })
     );
   });
+
+  it("should be exist button upload", () => {
```

Comment:
Maybe in this way?

```suggestion
  it("should have the upload button", () => {
```

(the generated sentence would be: _CustomMediaLibraryUI should have the upload button_)